PKGBUILD

```diff
@@ -31,8 +30,20 @@ depends=(
 )
 
 _dir="rqt-${pkgver}/rqt_gui"
-source=("${pkgname}-${pkgver}.tar.gz"::"https://github.com/ros-visualization/rqt/archive/${pkgver}.tar.gz")
-sha256sums=('3f62f90db254e111483b9a7cb9d486d0bf07c863b2a987808dc3636ee7004c22')
+source=(
+	python39.patch
```

Comment:
Can you add the patch from an upstream pull request? This is so that we make sure that the work that we do here is usable for the rest of the community. Essentially make an upstream pull request that fixes the issue and add it in like it is down here: 

https://github.com/ros-noetic-arch/ros-noetic-plotjuggler/blob/48da0824ef92fab5ab88b8fb612ffe581e38eb5e/PKGBUILD#L45-L48

Reply:
done

Reply:
Can you make a corresponding issue in this repo why the patch is needed? So that we can keep track of it. I know I might be asking a lot, but I just want to make sure we are following the conventions that we have been working with for the past couple of years for consistency as well as effectiveness. Thanks so much.

Reply:
@AchmadFathoni can you follow up on this and make a corresponding issue in this repo?

Reply:
Sure, the issue is in #2

Reply:
thx!